jdj_tags/extensions.py

```diff
@@ -263,12 +264,12 @@ def _make_blocktrans(self, singular, plural=None, context=None, trans_vars=None,
         else:
             if context is None:
                 return ungettext(
-                    force_text(singular), force_text(plural), trans_vars[count_var]
+                    force_text(singular), force_text(plural), int(trans_vars[count_var])
```

Comment:
I'm not sure I understand. count_var is just the name of the counter variable. How would you write this line?